lib/logstash/codecs/netflow.rb

```diff
@@ -57,10 +57,6 @@ def initialize(params = {})
     @threadsafe = true
   end
 
-  def clone
```

Comment:
There was an original reason for wanting to avoid copies of the codec https://github.com/logstash-plugins/logstash-codec-netflow/issues/122
This is because network devices may only send the template information about the flow once, and if we have multiple copies of the codec then some of the instances will never be able to decode certain flows, while others will.
From this I'd say the right fix would be to still override the method, but update the signature:

```ruby
def clone(*args)
  self
end
```